backend/tests/services/test_delivery_storage.py

```
@@ -0,0 +1,65 @@
+        "/deliveries/markdown.md",
+        "request-id",
+        "host-id",
+        bucket_name="deliveries",
+        object_name="deliveries/markdown.md",
+    )
+    storage = DeliveryStorage()
+    storage._client = client
+
+    with pytest.raises(
+        DeliveryObjectNotFoundError,
+        match="deliveries/markdown.md",
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)